src/importexport/guitarpro/internal/guitarbendimport/benddataprocessor.cpp

```diff
@@ -241,19 +241,18 @@ static void createTiedNotesBends(const BendDataContext& bendDataCtx, mu::engravi
                 continue;
             }
 
-            Chord* nextChord = chord->nextTiedChord();
-            if (!nextChord) {
-                continue;
-            }
-
             for (size_t noteIndex = 0; noteIndex < chord->notes().size(); noteIndex++) {
                 if (!muse::contains(tickInfo, noteIndex)) {
                     continue;
                 }
 
                 const auto& noteInfo = tickInfo.at(noteIndex);
-                Note* startNote = chord->notes()[noteIndex];
-                Note* endNote = nextChord->notes()[noteIndex];
+                Note* startNote = getLocatedNote(score, tick, track, noteIndex);
```

Comment:
Is there any reason the definition of `startNote` has to change? I'd think the old version does the same and avoids semi-expensive `getLocatedChord`.